query/language/define.feature

```diff
@@ -294,59 +294,59 @@ Feature: TypeQL Define Query
       | label:house |
 
 
-  Scenario: defining a type without a 'sub' clause throws
-    Then typeql define; throws exception
+  Scenario: defining a type without a kind throws
+    Then typeql define; fails
       """
       define flying-spaghetti-monster;
       """
 
 
-  Scenario: a type cannot directly subtype 'thing' itself
-    Then typeql define; throws exception
-      """
-      define column sub thing;
-      """
+#  Scenario: a type cannot directly subtype 'thing' itself
+#    Then typeql define; fails
+#      """
+#      define column sub thing;
+#      """
```

Comment:
This can become "a type definition must provide kind": `define column;` fails.
Otherwise, delete.